omnigibson/robots/fetch.py

```diff
@@ -501,3 +498,17 @@ def arm_workspace_range(self):
         return {
             self.default_arm : [np.deg2rad(-45), np.deg2rad(45)]
         }
+    
+    @property
+    def eef_usd_path(self):
+        return {self.default_arm: os.path.join(gm.ASSET_PATH, "models/fetch/fetch/fetch_eef.usd")}
+
+    @property
+    def teleop_rotation_offset(self):
+        return {self.default_arm: euler2quat([0, np.pi / 2, np.pi])}
+    
+    def teleop_data_to_action(self, teleop_data: dict):
+        action = np.zeros(self.action_dim)
+        action[:2] = TwoWheelRobot.teleop_data_to_action(self, teleop_data)
+        action[4:] = ManipulationRobot.teleop_data_to_action(self, teleop_data)
```

Comment:
hmm interesting, this doesn't seem very pythonic, but I am not sure what's the best way to do this. any ideas? @cremebrule @cgokmen

Reply:
check if `super().teleop_data_to_action` works

Reply:
confirmed works